app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import OutdatedVaccineError
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError

```

Comment:
You can join these `import` statements to one
```suggestion
from app.errors import (
    OutdatedVaccineError,
    NotVaccinatedError,
    NotWearingMaskError,
)
```